src/patternfly/components/List/list.scss

```diff
@@ -13,7 +13,7 @@
   --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
   --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
   --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
```

Comment:
This doesn't match our naming scheme. It should be `--pf-global--ListStyle` It's a typo in _variable.scss instead of here.

Reply:
This is resolved in https://github.com/patternfly/patternfly-next/pull/1048 btw